src/index.ts

```
@@ -208,11 +216,15 @@ export { Quiz };
+export { QuizQuestionList };
+export { QuizFooter };
+export { QuizListResult };
 export { QuizResultHeaderTitle };
 export { QuizResultTitle };
 export { QuizResultPerformance };
 export { QuizListResultByMateria };
-export { QuizQuestionList };
-export { QuizFooter };
-export { useQuizStore };
+
+export { MultipleChoiceList };
```

ESLint

[error] Insert `⏎`

(prettier/prettier)